Helper/EndpointHealthCheckTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+namespace Divido\DividoFinancing\Helper;
+
+
+use PHPUnit\Util\Exception;
+
+trait EndpointHealthCheckTrait
+{
+    /**
+     * Uses health endpoint result from SDK
```

Comment:
```suggestion
     * Returns true if the endpoint is healthy
     It useds the SDK health handler to determine the health status of the endpoint
```

something around this lines